src/Compilers/CSharp/Portable/Binder/Semantics/Conversions/ConversionsBase.cs

```diff
@@ -903,9 +914,38 @@ private static bool HasImplicitNullableConversion(TypeSymbol source, TypeSymbol
                 return true;
             }
 
+            if (HasImplicitTupleConversion(unwrappedSource, unwrappedDestination, ref useSiteDiagnostics))
+            {
+                return true;
+            }
+
             return false;
         }
 
+        private bool HasImplicitTupleConversion(TypeSymbol source, TypeSymbol destination, ref HashSet<DiagnosticInfo> useSiteDiagnostics)
+        {
+            ImmutableArray<TypeSymbol> sourceTypes;
+            ImmutableArray<TypeSymbol> destTypes;
+
+            if (!source.TryGetElementTypesIfTupleOrCompatible(out sourceTypes) ||
```

Comment:
Should also have tests for IMethodSymbol IMethodSymbol.ReduceExtensionMethod(ITypeSymbol receiverType), including cases when tuple types have the same element types, but are based on different underlying types.

---
In reply to: [66189636](https://github.com/dotnet/roslyn/pull/11681#discussion_r66189636) [](ancestors = 66189636,66189168)